guide/text/15-images.md

```diff
@@ -10,6 +10,10 @@ title: Images
 
 {% raw %}
 
+## Adding image files
+
+Our template comes with four folders for images, which correspond to output formats: `print-pdf`, `screen-pdf`, `web` and `epub`. Save your images files there. Each folder should contain the same set of images, but formatted for each format. For instance, while their file names must be identical, `web` images might be full-colour, 96dpi, and up to 800 pixels wide; while `print-pdf` images might be in greyscale, 300dpi and 2400 pixels wide.
```

Comment:
"formatted for each format" feels a little clunky?

"saved appropriately for each format" maybe?